samples/bookinfo/platform/kube/bookinfo-db.yaml

```diff
@@ -49,7 +49,7 @@ spec:
     spec:
       containers:
       - name: mongodb
-        image: quay.io/maistra/examples-bookinfo-mongodb:2.1.0
+        image: quay.io/maistra/examples-bookinfo-mongodb:2.1-z
```

Comment:
This and below changes don't look right.

Reply:
These are for the IBM system Z.  If we want to have a generic set of bookinfo images for Z, P and x86, we should federate them.  Then 2.1.0 would be correct. As it is now, 2.1-z is just as correct as 2.1.0, since both need to be modified for one platform or another.